solr/8.11.1.wso2v6/pom.xml

```diff
@@ -0,0 +1,253 @@
+    </distributionManagement>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <extensions>true</extensions>
+                <version>${maven.bundle.plugin.version}</version>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Bundle-Name>${project.artifactId}</Bundle-Name>
+                        <Export-Package>
+                            org.apache.solr.*;version="${version.solr}"
+                        </Export-Package>
+                        <Import-Package>
+                            !org.apache.solr.*,
+                            !org.eclipse.jetty.*,
+                            org.apache.lucene.*; version="${version.lucene}",
+                            org.apache.commons.io.*;version="${version.commons.io}",
+                            org.noggit.*;version="${version.noggit}",
+                            com.spatial4j.*;version="${version.spatial4j}",
+                            org.restlet.*;version="${version.restlet}",
+                            org.locationtech.spatial4j.*; version="${version.locationtech.spatial4j}",
+                        </Import-Package>
+                        <Private-Package></Private-Package>
+                        <Embed-Dependency>
+                            solr-solrj;scope=compile|runtime;inline=true,
+                            rrd4j;scope=compile|runtime;inline=false,
+                            jetty-util;scope=compile|runtime;inline=false,
+                            jetty-io;scope=compile|runtime;inline=false,
+                            jetty-http;scope=compile|runtime;inline=false,
+                            jetty-client;scope=compile|runtime;inline=false,
+                            http2-http-client-transport;scope=compile|runtime;inline=false,
+                            http2-common;scope=compile|runtime;inline=false,
+                            http2-client;scope=compile|runtime;inline=false,
+                            calcite-linq4j;scope=compile|runtime;inline=false,
+                            calcite-core;scope=compile|runtime;inline=false,
+                            avatica-core;scope=compile|runtime;inline=false,
+                            commons-collections4;scope=compile|runtime;inline=false,
+                            caffeine;scope=compile|runtime;inline=false,
+                            zookeeper;scope=compile|runtime;inline=false,
+                            metrics-core;scope=compile|runtime;inline=false,
+                        </Embed-Dependency>
+                        <Include-Resource>@solr-core-${version.solr}.jar!/*</Include-Resource>
+                        <!--Dynamic imports enabled here because solr uses dynamic class loading to import lucene -->
```

Comment:
Add a space before starting the comment, and finish the comment with a full stop.

Reply:
Fixed in https://github.com/wso2/orbit/pull/837